README.md

```diff
@@ -241,157 +188,46 @@ export function middleware(req: NextRequest) {
   getCookie('test', { res, req });
   getCookies({ res, req });
 
-  ❌ setCookie('test', 'value', { cookies }); 👉🏻// Won't work.
```

Comment:
In my opinion, this line should stay as it is.

Reply:
kept! Thanks!